RUMM-943 Update the docs following the new API for SDK initialisation (#459)
DataDog/dd-sdk-android: docs/sdk_initialisation.md

> #SDK Initialisation

Reviewer: ```suggestion
#SDK Initialization
```
And you should change the filename to sdk_initialization.md.

    
    {{< tabs >}}
    {{% tab "US" %}} 

```kotlin
class SampleApplication : Application() {
    override fun onCreate() {
        super.onCreate()

        val configuration = Configuration.Builder().build()
        val credentials = Credentials(
           <CLIENT_TOKEN>,
           <ENV_NAME>,
           <APP_VARIANT_NAME>,
           <APPLICATION_ID>
        )
        Datadog.initialize(this, credentials, configuration, trackingConsent)
    }
}
```
    {{% /tab %}}
    {{% tab "EU" %}}

```kotlin
class SampleApplication : Application() {
   override fun onCreate() {
      super.onCreate()
      
      val configuration = Configuration.Builder()
         .useEUEndpoints()
         .build()
      val credentials = Credentials(
         <CLIENT_TOKEN>,
      <ENV_NAME>,
      <APP_VARIANT_NAME>,
      <APPLICATION_ID>
      )
      Datadog.initialize(this, credentials, configuration, trackingConsent)
   }
```

    {{% /tab %}}
    {{< /tabs >}}

> You could notice that in the credentials required for initilisation we also require your application variant name. This is very important for us as based on it we will
> be able to automatically upload the right proguard `mapping.txt` file at build time. This will be needed to automatically de-obfuscate the stack traces in the Datadog dashboard.

Reviewer: ```suggestion
Note that in the credentials required for initialization, your application variant name is also required. This is important because it enables  the right proguard `mapping.txt` file to be automatically uploaded at build time. This allows a Datadog dashboard to de-obfuscate the stack traces.
```

> There is also a utility method which gives you the current state of the SDK. You can use this to check if it was properly initialized or not:

Reviewer: same comment as above

```kotlin
if(Datadog.isInitialized()){
  // your code here
}
```

> When writing your application, you can enable development logs. All internal messages in the library with a priority equal  or higher than the provided level are then logged to Android's Logcat.

Reviewer: same comment as above
